airflow/providers/google/cloud/transfers/gcs_to_sftp.py

```diff
@@ -52,6 +52,9 @@ class GCSToSFTPOperator(BaseOperator):
     :param destination_path: The sftp remote path. This is the specified directory path for
         uploading to the SFTP server.
     :type destination_path: str
+    :param ignore_gcs_path_in_destination: (Optional) When set to False the path of the file
```

Comment:
It confused me a little bit. Can you explain this parameter?

Reply:
For example: If you want to copy file from bucket: `gs://bucket_name/data/2020/file.txt` to `/tmp` directory, by default it will be copied reflecting object path on the bucket, so file path on sftp will be `/tmp/data/2020/file.txt`. If parameter `ignore_gcs_path_in_destination` will be set to True the file path on sftp will be `/tmp/file.txt`.

I am open for proposals for better name for this parameter :)

Reply:
Got it. I do not have better solution.

Reply:
Can you add an example to the description?

Reply:
Sure. Example was added.